manifests/02_operator_cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
+kind: CSISnapshotController
+metadata:
+  name: cluster
+  annotations:
+    release.openshift.io/create-only: "true"
+spec:
+  managementState: Managed
```

Comment:
Does this needs to have a namespace?

Reply:
No, CSISnapshotController is not namespaced.